internal/projects/delete.go

```diff
@@ -38,7 +38,7 @@ func CleanUpUnmanagedProjects(
 	proj uuid.UUID,
 	querier db.Querier,
 	authzClient authz.Client,
-	providerService service.GitHubProviderService,
+	providerManager manager.ProviderManager,
```

Comment:
In future, I might investigate creating a `ProjectService` or similar - right now we end up having to pass around a bunch of dependencies through the controlplane for the project creation logic.